app/services/audit_results.rb

```diff
@@ -216,8 +215,12 @@ def log_msg_prefix
     @log_msg_prefix ||= "#{check_name}(#{druid}, #{moab_storage_root&.name})"
   end
 
-  def workflows_msg_prefix
-    @workflows_msg_prefix ||= begin
+  def results_as_string(error_results)
+    @results_as_string ||= "#{string_prefix} #{error_results.map(&:values).flatten.join(' && ')}"
```

Comment:
i am hesitant to do this bit of caching.  in practice, i think it's fine as its used now.  but then it also means that this method can't return the right answer if results are ever added to the `AuditResults` instance after the first call to `#results_as_string`.  on the one hand, i don't know that i expect us to start adding audit results to an instance after it's been told to report -- that's not our current usage, AFAIK.  on the other hand, i wouldn't really expect to see much performance benefit from this caching -- since we're only calling it once in the current code, the caching isn't actually being used, and i don't think a significant fraction of pres cat's computational work happens in audit results reporting.  as such, this doesn't feel worth the complexity or possible cognitive overhead, to me at least.

this seems a bit different than the `@string_prefix` caching below -- all the things used in the string built by `#string_prefix` are known at constructor invocation, and aren't mutatable via the public methods of this class.

Reply:
Note:  it is called twice because the PR I am working on to populate the data in the new field uses this method as well.

Would you rather I:
1.  added a comment to this method saying it was expected to be used after results are stable
2.  took out the caching

I don't have strong feelings.

Reply:
ah, makes sense in light of the follow on work.

slight preference for option 2 (just taking out the caching), but also totally fine w/ a comment.  up to you!

Reply:
no longer cached.